### ISSUE TITLE
join-testnet.sh axelar-core panic after removing data directory

### ISSUE THREAD

**Issue by ggutoski** [axelarnetwork/axelarate-community#281]
**Describe the bug**
axelar-core panic running `join-testnet.sh` on current testnet (toronto).  Seems from the panic trace to be an unmarshalling error in the genesis file.  Are we pulling the wrong genesis file?

**To Reproduce**
1. Remove the `-d` and `-rm` flags from the `docker run` command in `join-testnet.sh` so you can see the logs.
2. Start with a fully synced and running validator node.  (Probably don't need it to be a validator or fully synced.)
3. Stop the node `docker stop vald tofnd axelar-core`
4. remove all data: `mv ~/.axelar_testnet ~/.axelar_testnet_backup`
5. Start the node again `join-testnet.sh`

Container panics on start-up.  Snippet of output.  (I have the whole output in a text file.)
```
2021-12-28T07:06:17Z INF port binded module=x/ibc/port port=transfer
2021-12-28T07:06:17Z INF claimed capability capability=1 module=transfer name=ports/transfer
panic: json: cannot unmarshal object into Go value of type string

goroutine 1 [running]:
github.com/cosmos/cosmos-sdk/codec.(*ProtoCodec).MustUnmarshalJSON(0xc0012f6530, 0xc000c20780, 0x1cc, 0x1e0, 0x27bbf80, 0xc0012d4d00)
	/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.44.3/codec/proto_codec.go:176 +0x98
github.com/axelarnetwork/axelar-core/x/axelarnet.AppModule.InitGenesis(0x27d6840, 0xc0012c58c0, 0x27b47f0, 0xc000f487b0, 0x27ef9c8, 0xc0012f6530, 0x27ef9c8, 0xc0012f6530, 0xc00120e060, 0x27b47f0, ...)
	/go/axelar/x/axelarnet/module.go:129 +0xa4
github.com/cosmos/cosmos-sdk/types/module.(*Manager).InitGenesis(0xc001117dc0, 0x27d5848, 0xc000138010, 0x27f2ba0, 0xc0001fad00, 0x0, 0x0, 0xc0012f0030, 0x15, 0x0, ...)
	/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.44.3/types/module/module.go:307 +0x2b5
github.com/axelarnetwork/axelar-core/app.(*AxelarApp).InitChainer(0xc000f3b200, 0x27d5848, 0xc000138010, 0x27f2ba0, 0xc0001fad00, 0x0, 0x0, 0xc0012f0030, 0x15, 0x0, ...)
	/go/axelar/app/app.go:576 +0x170
github.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).InitChain(0xc0000ae1a0, 0x38e4cdd8, 0xed95c23f3, 0x0, 0xc0012f0030, 0x15, 0xc000142160, 0x3742dc0, 0x0, 0x0, ...)
	/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.44.3/baseapp/abci.go:62 +0x315
github.com/tendermint/tendermint/abci/client.(*localClient).InitChainSync(0xc0006a2600, 0x38e4cdd8, 0xed95c23f3, 0x0, 0xc0012f0030, 0x15, 0xc000142160, 0x3742dc0, 0x0, 0x0, ...)
	/go/pkg/mod/github.com/tendermint/tendermint@v0.34.14/abci/client/local_client.go:272 +0x115
github.com/tendermint/tendermint/proxy.(*appConnConsensus).InitChainSync(0xc000f78ea0, 0x38e4cdd8, 0xed95c23f3, 0x0, 0xc0012f0030, 0x15, 0xc000142160, 0x3742dc0, 0x0, 0x0, ...)
	/go/pkg/mod/github.com/tendermint/tendermint@v0.34.14/proxy/app_conn.go:77 +0x78
github.com/tendermint/tendermint/consensus.(*Handshaker).ReplayBlocks(0xc0027bda48, 0xb, 0x0, 0x2298548, 0x7, 0xc0012f0030, 0x15, 0x1, 0x0, 0x0, ...)
	/go/pkg/mod/github.com/tendermint/tendermint@v0.34.14/consensus/replay.go:319 +0x655
github.com/tendermint/tendermint/consensus.(*Handshaker).Handshake(0xc0027bda48, 0x27f44d8, 0xc0010fd930, 0x1f4, 0xc000e86480)
	/go/pkg/mod/github.com/tendermint/tendermint@v0.34.14/consensus/replay.go:268 +0x458
github.com/tendermint/tendermint/node.doHandshake(0x27ef490, 0xc000f78260, 0xb, 0x0, 0x2298548, 0x7, 0xc0012f0030, 0x15, 0x1, 0x0, ...)
	/go/pkg/mod/github.com/tendermint/tendermint@v0.34.14/node/node.go:325 +0x1d8
github.com/tendermint/tendermint/node.NewNode(0xc000c3a8c0, 0x27cb610, 0xc0012dfb80, 0xc00051a010, 0x278ca40, 0xc0005203a8, 0xc00051a0f0, 0x25322e8, 0xc00051a270, 0x27d6840, ...)
	/go/pkg/mod/github.com/tendermint/tendermint@v0.34.14/node/node.go:733 +0x2105
github.com/cosmos/cosmos-sdk/server.startInProcess(0xc000194100, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x27e0dd0, 0xc0011157b0, ...)
	/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.44.3/server/start.go:263 +0x685
github.com/cosmos/cosmos-sdk/server.StartCmd.func2(0xc0011fe780, 0x3742dc0, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.44.3/server/start.go:129 +0x209
github.com/spf13/cobra.(*Command).execute(0xc0011fe780, 0x3742dc0, 0x0, 0x0, 0xc0011fe780, 0x3742dc0)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc00110f900, 0x0, 0x0, 0xc00107bf70)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:895
github.com/cosmos/cosmos-sdk/server/cmd.Execute(0xc00110f900, 0xc00107bf70, 0xd, 0x4000000000000000, 0x0)
	/go/pkg/mod/github.com/cosmos/cosmos-sdk@v0.44.3/server/cmd/execute.go:36 +0x285
main.main()
	/go/axelar/cmd/axelard/main.go:70 +0x155
```

**Expected behavior**

I expect `join-testnet.sh` to join the testnet and start syncing the chain from scratch as if it were a fresh new node.

**Additional context**
Why did I do this?  I want to test validator recovery flows.  My first attempt also included mnemonic recovery flags for `join-testnet.sh` and hit the same panic.  I soon discovered that the panic occurs even without the recovery flags.

Same result even if you include `--reset-chain`.  The only way I can get my node to run is if I restore the backup I made above

**Comment by ggutoski** [comment]
no longer relevant after #289
